docs/manual/cluster-admin/installation-guide.md

```diff
@@ -192,20 +322,41 @@ The `user` and `password` is the SSH username and password from dev box machine
 
 ### Start Kubernetes
 
-On the dev box machine, use the following commands to clone the OpenPAI repo:
+Go to folder `<pai-code-dir>/contrib/kubespray`:
 
 ``` bash
-git clone -b pai-1.4.y https://github.com/microsoft/pai.git # change to a different branch if you want to deploy a different version
-cd pai/contrib/kubespray
+cd <pai-code-dir>/contrib/kubespray
 ```
 
-The folder `pai/contrib/kubespray` contains installation scripts, both for kubespray and OpenPAI services.
-Please run the following script to deploy Kubernetes first:
+The folder `pai/contrib/kubespray` contains installation scripts, both for Kubernetes and OpenPAI services.
+Please run the following script to deploy Kubernetes first. As the name explains, we adopt [kubespray](https://github.com/kubernetes-sigs/kubespray) to install Kubernetes.
 
 ``` bash
 /bin/bash quick-start-kubespray.sh
 ```
 
+If there is any problem, please double check the environment requirements first. Here we provide a requirement checker to help you verify:
+
+``` bash
+/bin/bash requirement.sh
+```
```

Comment:
-l ${LAYOUT} -c ${CLUSTER_CONFIG} are needed for requirement.sh

Reply:
Done